AirCasting/Map/Heatmap/GridSquare.swift

```diff
@@ -47,8 +55,7 @@ struct GridSquare {
         number += 1
         calculateAverage()
         
-        guard let averagedValue = averagedValue else { return }
-        let color: UIColor = GoogleMapView.color(value: Int32(averagedValue), threshold: sensorThreshold).withAlphaComponent(0.5)
+        let color: UIColor = GoogleMapView.color(value: properThreshold, threshold: sensorThreshold).withAlphaComponent(0.5)
```

Comment:
As noted above, `properThreshold` should be renamed to reflect what it actually is, and I think we should keep it as an optional and keep the guard, instead of passing 0 forward.